python/pyspark/sql/connect/streaming/worker/foreach_batch_worker.py

```diff
@@ -63,8 +63,13 @@ def main(infile: IO, outfile: IO) -> None:
     spark = spark_connect_session
 
     # TODO(SPARK-44461): Enable Process Isolation
+    try:
```

Comment:
Can this try be around teh while loop below? I.e
```
try:
     func = worker.read_command(pickle_ser, infile)
     write_int...
     while True:
         ..
except
```
Note that currently, errors around `read_long()` are not handled well.